accessibility-oscilloscope/overlay/usr/share/test.d/S03-pnt-lut.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+killall pnt-lut &>/dev/null || true
+
+hex_string() {
+    echo $1 | xxd -r -p
+}
+
+mkfifo /tmp/pgm
+mkfifo /tmp/pnt
+
+pnt-lut /tmp/pgm /tmp/pnt /dev/stdout | od -w2 -x | tee /tmp/haptic &
+cat /usr/share/data/gradient-x.pgm >/tmp/pgm
+hex_string "00 10 00 00 00 00 00 00" >/tmp/pnt
+hex_string "00 20 00 00 00 00 00 00" >/tmp/pnt
+hex_string "00 30 00 00 00 00 00 00" >/tmp/pnt
+hex_string "00 40 00 00 00 00 00 00" >/tmp/pnt
+killall pnt-lut
```

Comment:
there's a really fun race condition here. when `pnt-lut` is instatiated (in the background) with the redirect, `killall` may send `SIGKILL` before the pipe finishes writing.

i'm not sure how to test deterministically but the actual program being tested works fine.